### PR TITLE
ST-3073 fix root path

### PR DESCRIPTION
**Pull request by daraosn**

### CODE REVIEW

File: styla.php [hunk 1]
```diff
@@ -252,9 +252,9 @@ private function buildContentKey() {
             $contentKey = isset($_REQUEST["origUrl"]) ? $_REQUEST["origUrl"] : "/";
 
             if($this->rootPath !== "/"){
-                $path = join([ "/", $this->rootPath ]);
```
Comment (daraosn) [inline styla.php:255]: @ssachtleben is the `rootPath` param always a string? I am assuming so for the fix.

File: index.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
     include("styla.php");
 
     $styla = new Styla('showcase', [
-        "rootpath" => "/"
+        "rootPath" => "/"
```
Comment (daraosn) [inline index.php:5]: 1. first issue was that the array key is case senstive, so the styla.php was receiving a NULL string

File: styla.php [hunk 3]
```diff
@@ -252,9 +252,9 @@ private function buildContentKey() {
             $contentKey = isset($_REQUEST["origUrl"]) ? $_REQUEST["origUrl"] : "/";
 
             if($this->rootPath !== "/"){
```
Comment (daraosn) [inline styla.php:254]: 2. second issue was that as the rootPath was being passed as NULL, this condition evaluated to true, so it was defining `$path` as `/`, deleting all the slashes on the url, so the SEO API was returning a 404 in all cases except for root url.

File: styla.php (hunk 1 above)
Comment (ssachtleben) [reply styla.php:255]: yes

**Review by ssachtleben — APPROVED** [review]

**Pull request merged**